### PR TITLE
pmount: new package at 0.9.23

### PR DESCRIPTION
**Pull request by ilmich**
## Description of package
pmount ("policy mount") is a wrapper around the standard mount program which permits normal users to mount removable devices without a matching /etc/fstab entry. 

## Installed manifest of package
(`kiss manifest pmount`)

```
/var/db/kiss/installed/pmount/version
/var/db/kiss/installed/pmount/sources
/var/db/kiss/installed/pmount/patches/001-fix-musl-build.patch
/var/db/kiss/installed/pmount/patches/001-disable-intltool.patch
/var/db/kiss/installed/pmount/patches/
/var/db/kiss/installed/pmount/manifest
/var/db/kiss/installed/pmount/depends
/var/db/kiss/installed/pmount/checksums
/var/db/kiss/installed/pmount/build
/var/db/kiss/installed/pmount/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man1/pumount.1
/usr/share/man/man1/pmount.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/
/usr/bin/pumount
/usr/bin/pmount
/usr/bin/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify the @kiss-community maintainers if I can no longer maintain this package.

### CODE REVIEW

File: community/pmount/build [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/sh -e
+
+patch -p1 < 001-disable-intltool.patch
+patch -p1 < 001-fix-musl-build.patch
+
+# autoconf warnings are treated as errors.
+set +e
+automake
+autoconf
+set -e
```
Comment (dilyn-corner) [inline community/pmount/build:10]: We should never really set +e. If autotools wants to be broken, we should let it :X 
`autoreconf -sfi` is what we should do instead, though presumably this is responsible for me having to modify the `Makefile` after it is generated.

File: community/pmount/build [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+./configure \
+    --prefix=/usr \
+    --disable-nls \
```
Comment (dilyn-corner) [inline community/pmount/build:14]: `--disable-nls` is no longer a flag after we remove the `intltool` dep

File: community/pmount/build [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    --with-media-dir=/mnt/
+
+make
+
+# 'make install' requires root privileges.
+install -Dm4755 src/pmount "$1/usr/bin/pmount"
+install -Dm4755 src/pumount "$1/usr/bin/pumount"
+install -Dm0644 man/pmount.1 "$1/usr/share/man/man1/pmount.1"
+install -Dm0644 man/pumount.1 "$1/usr/share/man/man1/pumount.1"
```
Comment (dilyn-corner) [inline community/pmount/build:23]:
```
install -Dm4755 src/pmount    "$1/usr/bin/pmount"
install -Dm4755 src/pumount   "$1/usr/bin/pumount"
install -Dm0644 man/pmount.1  "$1/usr/share/man/man1/pmount.1"
install -Dm0644 man/pumount.1 "$1/usr/share/man/man1/pumount.1"
```

File: community/pmount/build [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+automake
+autoconf
+set -e
+
+./configure \
+    --prefix=/usr \
+    --disable-nls \
+    --with-media-dir=/mnt/
+
+make
```
Comment (dilyn-corner) [inline community/pmount/build:17]: Without running `automake && autoconf` and instead using `autoreconf -sfi`, you'd have to add
`sed -i 's/po src tests/src/' Makefile` before `make`.

**Review by dilyn-corner — COMMENTED** [review]
Why is autotools so gross. 

You'll need a `depends` file specifying that `automake` and `util-linux` are deps (`pmount` requires `blkid`).

If there's a way to not need to use a `sed` *after* configure without setting +e that would be nice to have; I haven't found it yet. Their `configure` script is a bit insistent on `intltool` :\

If you could also remove the irrelevant commits :)

**Comment by ilmich** [comment]
> Why is autotools so gross.

I agree, unfortunately pmount is too comfortable for me.

> If you could also remove the irrelevant commits :)

before I start, my git-flow (maybe buggy) is the following
```
$ git fetch upstream
$ git checkout main
$ git merge upstream/main
$ git branch -b <package>
$ .....
$ .....
$ .....
$ git push origin <package>
```
so I don't know what went wrong (libdvdread is already merged)

If you prefer I can reset my repo (but I would like to fix it)

**Comment by dilyn-corner** [comment]
I use rebase instead of merge. 

If you rebase this branch onto upstream/main and then force push it should probably clean it up

**Comment by ilmich** [comment]
All fixes are in place.

Thanks!!

File: community/pmount/depends [hunk 5]
```diff
@@ -0,0 +1,3 @@
+autoconf make
+util-linux
+
```
Comment (dilyn-corner) [inline community/pmount/depends:3]: `autoconf` should instead be `automake` as `autoreconf` will use `aclocal`. 
Remove the blank line at the end of this.

**Review by dilyn-corner — COMMENTED** [review]
Sorry for the delay; thank you for the fixes! Just two small nits and it'll be all set :)

**Comment by ilmich** [comment]
> Sorry for the delay;
 
Don't worry, we are all volunteers ;)

>thank you for the fixes! Just two small nits and it'll be all set :)

fixed. Let me known.

**Pull request merged**